houston/workspace_teams.go

```diff
@@ -0,0 +1,76 @@
+package houston
+
+// AddTeamToWorkspace - add a team to a workspace
+func (h ClientImplementation) AddWorkspaceTeam(workspaceUUID, teamUuid, role string) (*Workspace, error) {
+	req := Request{
+		Query:     WorkspaceTeamAddRequest,
+		Variables: map[string]interface{}{"workspaceUuid": workspaceUUID, "teamUuid": teamUuid, "role": role},
+	}
+
+	r, err := req.DoWithClient(h.client)
+	if err != nil {
+		return nil, handleAPIErr(err)
+	}
+
+	return r.Data.AddWorkspaceTeam, nil
+}
+
+// RemoveTeamFromWorkspace - remove a team from a workspace
+func (h ClientImplementation) DeleteWorkspaceTeam(workspaceUUID, teamUuid string) (*Workspace, error) {
+	req := Request{
+		Query:     WorkspaceTeamRemoveRequest,
+		Variables: map[string]interface{}{"workspaceUuid": workspaceUUID, "teamUuid": teamUuid},
+	}
+
+	r, err := req.DoWithClient(h.client)
+	if err != nil {
+		return nil, handleAPIErr(err)
+	}
+
+	return r.Data.RemoveWorkspaceTeam, nil
+}
+
+// ListTeamAndRolesFromWorkspace - list teams and roles from a workspace
+func (h ClientImplementation) ListWorkspaceTeamsAndRoles(workspaceUUID string) ([]Team, error) {
+	req := Request{
+		Query:     WorkspaceGetTeamsRequest,
+		Variables: map[string]interface{}{"workspaceUuid": workspaceUUID},
+	}
+
+	r, err := req.DoWithClient(h.client)
+	if err != nil {
+		return nil, handleAPIErr(err)
+	}
+
+	return r.Data.WorkspaceGetTeams, nil
+}
+
+// UpdateTeamRoleInWorkspace - update a team role in a workspace
+func (h ClientImplementation) UpdateWorkspaceTeamRole(workspaceUUID, teamUuid, role string) (string, error) {
```

Comment:
```suggestion
func (c ClientImplementation) UpdateWorkspaceTeamRole(workspaceUUID, teamUuid, role string) (string, error) {
```
typically it's first latter of type.

Reply:
In a separate comment I asked to get all the UUID variables moved to ID since we use this term everywhere in the code base

Reply:
@andriisoldatenko @AmFlint I still see UUID used across files outside this PR. I am going to create a separate PR for updating the rest. I will follow UUID -> ID convention specific to files in this PR for now

Reply:
We are currently merging this Software CLI to the cloud CLI, and we plan on doing the refactoring at this moment, so do not worry about it for now

Reply:
done